src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -83,31 +83,89 @@ class PluginTableContainer:
     columns = attr.ib(default=attr.Factory(dict))
 
 
-@attr.s(frozen=True, slots=True)
-class TrendOutputDescription:
+@attr.s(kw_only=True)
+class _BaseTrendOutputDescription:
+    curve_names: List[str] = attr.ib(validator=list_of_strings)
+
+
+@attr.s()
+class PositionalTrendDescription(_BaseTrendOutputDescription):
```

Comment:
Right now both edge and equipment are considered "Positional" ones (listed on the same table). Maybe we could rename this one to `PositionalPipeTrendDescription` (since the other pipe related is `OverallPipeTrendDescription`)

Reply:
Good point, I'll change to `PositionalPipeTrendDescription`.